.gitmodules

```diff
@@ -51,6 +51,10 @@
 	path = sources/pmm-dump
 	url = https://github.com/percona/pmm-dump
 	branch = v3
+[submodule "watchtower"]
```

Comment:
I think we need to add it to `ci-default.yml` as well, at least until we merge [this PR](https://github.com/Percona-Lab/pmm-submodules/pull/3774).